pkg/network/tracer/tracer_test.go

```diff
@@ -1150,7 +1151,7 @@ func (s *TracerSuite) TestUnconnectedUDPSendIPv4() {
 	cfg := testConfig()
 	tr := setupTracer(t, cfg)
 
-	remotePort := rand.Int()%5000 + 15000
+	remotePort := testutil.GetOpenPort(t)
```

Comment:
`GetOpenPort` finds an open TCP port, but you're using it here in a UDP addr. You'll probably want to create another helper for UDP.